CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.9.8 - 2019-03-12
```

Comment:
This should be a 2.0.0 release, since removing those files is backwards-incompatible, according to @underscorgan.

Reply:
ok